public/app/features/alerting/unified/components/receivers/TemplateForm.tsx

```diff
@@ -101,6 +105,11 @@ export const TemplateForm = ({ existing, alertManagerSourceName, config, provena
   const [payload, setPayload] = useState(defaultPayloadString);
   const [payloadFormatError, setPayloadFormatError] = useState<string | null>(null);
 
+  const isProvisioned = Boolean(originalTemplate?.provenance) && originalTemplate?.provenance !== PROVENANCE_NONE;
```

Comment:
Would be nice to have this in the hook as well, and included as a prop on the template (then we can eventually use the same helper method there as well to check if its provisioned)

Reply:
I added a new `useNotificationTemplateMetadata` hook. For now it only provides `isProvisioned` value but we might extend it in the future